media/css/products/vpn/platforms/shared.scss

```diff
@@ -0,0 +1,437 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../../protocol/css/includes/lib';
+
+//* -------------------------------------------------------------------------- */
```

Comment:
There's a fair bit of stuff in here that's only related to the main landing page. Worth removing some of the leftovers?

Reply:
Removed 130 lines! 🙌